ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1219,6 +1219,17 @@ literals:
 - thermal_energy_accumulator
 - ground_isolation_command
 - ground_temperature_sensor
+
+
+#telemetry for PLC#
+- controller_run_command:
```

Comment:
These are not used elsewhere in this PR. Please revert.

Reply:
Deleted from telemetry fields...